src/components/MotionGenerator.tsx

```diff
@@ -22,12 +29,17 @@ const MotionGenerator = () => {
 
   return (
     <div className="flex flex-col items-center text-center">
-      <section className="max-w-[350px]">
+      <section className="flex flex-col space-y-2 max-w-[400px]">
```

Comment:
I see you using `space-{"x"|"y"}-{amount}` again and I originally thought - why not just flexbox gap? That's what it was made for.

But now that I think about it, it might actually work in places flex gap doesn't (like the smartboard on Fredry 13). Just need to check if it behaves the same as gap, because doing it with set margins seems prone to breaking.

🤔

Reply:
I can't tell if the difference between `space` and `gap` matters in this case. I won't insist on either of them though.

Reply:
I'll have to check if it's possible to apply space conditionally, if the browser doesn't support flexbox gap. Or, rather, if it's easy to do so, as it's definitely doable, just hinges on the workload needed to do so. There's a thing for that in CSS, question is if it supports properties in different contexts or just checks for its existence.